modules/components/stories/Treeview.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import ThemeSwitcher, { AVAILABLE_THEMES } from '../src/ThemeSwitcher';
+import NestedTreeView from '../src/NestedTreeView';
+import {
+  elasticMappingToDisplayTreeData,
+  MOCK_MAPPING,
+} from './mappingToTreeData';
+
+const dataSource = [
+  {
+    title: 'Animal',
+    id: 'animal',
+    children: [
+      {
+        title: 'Cat',
+        id: 'cat',
+      },
+      {
+        title: 'Dog',
+        id: 'dog',
+        children: [
+          {
+            title: 'Beagle',
+            id: 'beagle',
+            children: [
+              {
+                title: 'My Beagle',
+                id: 'beagle1',
+              },
+              {
+                title: "Alex's Beagle",
+                id: 'beagle2',
+              },
+            ],
+          },
+          {
+            title: 'Pug',
+            id: 'pug',
+            children: [
+              {
+                title: 'My Pug',
+                id: 'pug1',
+              },
+              {
+                title: "Alex's Pug",
+                id: 'pug2',
+              },
+            ],
+          },
+        ],
+      },
+    ],
+  },
+];
+
+storiesOf('Treeview', module).add('Treeview', () => (
+  <>
+    <ThemeSwitcher availableThemes={AVAILABLE_THEMES} />
```

Comment:
Could you use the theme decorator here? It recently got merged, you can look in `SQONView.js` for an example